### PR TITLE
K237 dev

### PR DESCRIPTION
**Pull request by A-D-Fuchs**
Added driver for the Keithley 237 SMU.

Small additions to visa_signal.py (process_read_function for processing read values for parsing) and device_class.py where label dictionary was added for auto generated config settings

### CODE REVIEW

File: bluesky_handling/visa_signal.py [hunk 1]
```diff
@@ -87,6 +87,7 @@ def __init__(self,  name, value=0., timestamp=None, parent=None, labels=None,
         #     self.visa_instrument.write_termination = write_termination
         #     self.visa_instrument.baud_rate = baud_rate
         self.read_function = read_function or None
+        self.process_read_function = process_read_function or None
```
Comment (Jo-Leh) [inline bluesky_handling/visa_signal.py:90]: I like this

**Review by Jo-Leh — CHANGES_REQUESTED** [review]
remove the files from SweepMe

**Comment by A-D-Fuchs** [comment]
Habs geändert glaub

**Review by Jo-Leh — APPROVED** [review]
very nice

**Pull request merged**